docs/assets/tags.json

```diff
@@ -29,7 +29,8 @@
     "kti": ["&9&Z[skyclient dev]&r", "&9&Z[SC dev]&r"],
     "alfredo": ["&7[Alfredo cuz my friend is eating it]&r", "&7[Alfredo]&r"],
     "Maya": ["&8[&dCutie&8]", "&8[&5Maya&8]"],
-    "Zetvue": ["&8[&c♫i make music♫&8]&r", "&8[&cZetvue&8]&r"]
+    "Zetvue": ["&8[&c♫i make music♫&8]&r", "&8[&cZetvue&8]&r"],
+    "Shinya": ["&5&l[&d&lMega Omega Weeb 6969 420SussyBalls Amogus&5&l]","&5&l[&d&lWeeb&5&l]"]
```

Comment:
do you find this "humorous"

Reply:
shinya on his way to make the most unfunny jokes anyone has ever heard

Reply:
uh, yes?
its fune